Description: Disable the antlr plugin for the tool module only. The generated source files are already in debian/generated-sources
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/tool/pom.xml
+++ b/tool/pom.xml
@@ -63,7 +63,7 @@
     <build>
 
         <plugins>
-
+<!--
             <plugin>
                 <groupId>org.antlr</groupId>
                 <artifactId>antlr3-maven-plugin</artifactId>
@@ -80,7 +80,7 @@
                 </executions>
 
             </plugin>
-
+-->
         </plugins>
 
 
